main/nestedtemplates/nic-pip-existing.json

```diff
@@ -41,6 +41,7 @@
             }
           }
         ],
+        "enableIPForwarding": true,
```

Comment:
Why we have to enable it for all the hosts? Lets user enable during deployment if he needed it.

Reply:
Implemented